src/sp/it/pl/main/App.kt

```diff
@@ -462,7 +462,7 @@ class App: Application(), Configurable<Any> {
 
     /** @return images of the icon of the application in all possible sizes */
     fun getIcons(): List<Image> = seqOf(16, 24, 32, 48, 128, 256, 512)
-            .map { File("icon$it.png").toURI().toString() }
+            .map { File("resources/icons/icon$it.png").toURI().toString() }
```

Comment:
I though it is a good idea to provide icons next to the exe/jar/launcher due to conventions. Thoughts on this?

Reply:
resources is a pretty self-explanatory folder, and I've often observed something like this or "assets" containing stuff like this. I personally belive that the top directory should be as clean as possible, the icons are still easily accessible

Reply:
I understand, but are you sure there are not conventions about where the application icon has to be and that sometimes it is expected to sit next to the exe/launcher for it to work properly? Thats the meaning here.

Reply:
No, I nearly never see that to be honest. Most often it is solely packaged _within_ the exe, or hidden somewhere.

Reply:
Ok.